### DISCUSSION TITLE
Welcome to go-chat Discussions!

### DISCUSSION THREAD

**Discussion by steelthedev** [steelthedev/go-chat#6] [Announcements]
Welcome to discussion

**Comment by devsujay19** [comment]
I saw your article on freecodecamp.org. Got to know this repository.

**Reply by steelthedev** [reply]
Nice! Been picking up some stuffs on Go recently. Thought it will be nice to write something about websockets, even though very simple stuff.

**Reply by devsujay19** [reply]
Yeah, Go's very simple to understand. Kind of JavaScript...

**Reply by devsujay19** [reply]
You're very good at writing documentation.

**Reply by steelthedev** [reply]
Yeah, actually I once worked as a technical writer for a writing platform. Nothing much, just writing articles and all.

**Reply by steelthedev** [reply]
While I dont actually find Javascript hard. I stay away from it. The JS ecosystem is just too hard to maneuver.

**Reply by devsujay19** [reply]
> Yeah, actually I once worked as a technical writer for a writing platform. Nothing much, just writing articles and all.

That's why, you wrote the README.md file with so perfection.

**Reply by devsujay19** [reply]
> While I dont actually find Javascript hard. I stay away from it. The JS ecosystem is just too hard to maneuver.

No, JavaScript isn't really hard. Now, you can build websites, desktop applications, terminal applications, console applications and mobile applications — all with JavaScript and its ecosystem.

**Reply by steelthedev** [reply]
This versatility makes Javascript more harder. The truth is Javascript does only the frontend better out of the box. The introduction of runtime like nodes and all made it too ambiguous.

I can write a full Backend service with Go without installing any dependencies but that cannot be said with Javascript. It's just too robust.

I can also write several APIs with python by installing very few dependencies both in development and production.

**Reply by devsujay19** [reply]
> This versatility makes Javascript more harder. The truth is Javascript does only the frontend better out of the box. The introduction of runtime like nodes and all made it too ambiguous.
> 
> I can write a full Backend service with Go without installing any dependencies but that cannot be said with Javascript. It's just too robust.
> 
> I can also write several APIs with python by installing very few dependencies both in development and production.

I think, we shouldn't argue. Because, we both are different developers with different tech stacks. So..... It's okay, what you're thinking about JavaScript. But from my perspective, it is a good language.

**Reply by steelthedev** [reply]
Actually, I agree that it is a good language. One that every developer might surely understand

**Comment by devsujay19** [comment]
Which pricing plan do you use on Render?

**Reply by steelthedev** [reply]
I use the free tier($0). I only host practice projects on render.


I get a server from linode or digital ocean whenever I want to host something worth it.

**Reply by devsujay19** [reply]
> I use the free tier($0). I only host practice projects on render.
> 
> I get a server from linode or digital ocean whenever I want to host something worth it.

Can I host unlimited projects on Render?
Actually, I'm new to Render. I use Vercel for deployment.

**Reply by steelthedev** [reply]
> .

Yes. You can host unlimited.

**Reply by devsujay19** [reply]
> > .
> 
> Yes. You can host unlimited.

Any project, like Python?

**Reply by steelthedev** [reply]
Yes, you can host python. Although I have only hosted flask and Django apps so far

**Reply by devsujay19** [reply]
That's all for me. Thanks for the information!

**Reply by devsujay19** [reply]
> Yes, you can host python. Although I have only hosted flask and Django apps so far

I hosted a chat application made with NodeJS and HTML. Deployed it to Render. Check it out here: https://socket-io-chat-app-dv2h.onrender.com.
